sw/host/opentitantool/src/command/certificate.rs

```diff
@@ -80,12 +81,29 @@ impl CommandDispatch for CodegenCommand {
         _context: &dyn Any,
         _transport: &TransportWrapper,
     ) -> Result<Option<Box<dyn Annotate>>> {
-        let template = load_template(&self.template)?;
+        let (template_name, codegen) = match self.cert_format {
+            CertFormat::X509 => {
+                let template = load_template(&self.template)?;
```

Comment:
maybe the `load_template` function should be moved to the `x509` module to be consistent with cwt?

Reply:
Currently the cwt implementation is like duplicating all the things from x509 to make a `cwt` version. I'm not sure if we move `load_template` into x509 module will make the situation better, since the `Template` itself is still tightly coupled with x509.